Next.js build is failing due to js-cookie being bundled as a dependency

**Describe the bug**
Next.js fails to compile a build due to `js-cookie` module.

```sh
./node_modules/@highlight-run/next/dist/next-client.js
Module not found: Can't resolve 'js-cookie'
```

**To Reproduce**
Steps to reproduce the behavior:

1. install in a next.js app `@highlight-run/next@latest` (I use yarn)
2. run `next build`
3. See error

```sh
Failed to compile.

./node_modules/@highlight-run/next/dist/next-client.js
Module not found: Can't resolve 'js-cookie'

https://nextjs.org/docs/messages/module-not-found


> Build failed because of webpack errors
error Command failed with exit code 1.
```

**Expected behavior**
Next build should pass without having to install `js-cookie` as a direct dependency to my project.

**Additional context**
I can see `js-cookie` being a devDependency in `@highlight-run/next` yet it's being bundled in the `dist/next-client.js` which actually means that it should be moved in the dependency section or it should be made a peerDependency (not ideal).

`dist/next-client.js`

```js
"use client";
"use strict";var e=require("react"),r=require("highlight.run"),t=require("js-cookie")...
//# sourceMappingURL=next-client.cjs.map
```

--- a/.changeset/fresh-spoons-compare.md
+++ b/.changeset/fresh-spoons-compare.md
@@ -1,0 +1,5 @@
+---
+'@highlight-run/next': patch
+---
+
+Moving js-cookie from devDependencies to dependencies

--- a/sdk/highlight-next/package.json
+++ b/sdk/highlight-next/package.json
@@ -63,7 +63,8 @@
 		"@highlight-run/opentelemetry-sdk-workers": "workspace:*",
 		"@highlight-run/react": "workspace:*",
 		"@highlight-run/sourcemap-uploader": "workspace:*",
-		"highlight.run": "workspace:*"
+		"highlight.run": "workspace:*",
+		"js-cookie": "^3.0.5"
 	},
 	"devDependencies": {
 		"@cloudflare/workers-types": "^4.20231016.0",
@@ -77,7 +78,6 @@
 		"chokidar-cli": "^3.0.0",
 		"eslint": "8.39.0",
 		"jest": "^29.2.0",
-		"js-cookie": "^3.0.5",
 		"rollup": "^4.4.1",
 		"tree-kill": "^1.2.2",
 		"ts-jest": "^29.0.3",

